internal/api/lib_test.go

```diff
@@ -15,8 +15,9 @@ import (
 	"github.com/CosmWasm/wasmvm/types"
 )
 
+var TESTING_FEATURES = "staking,stargate,iterator," + GetDefaultFeatures()
+
 const (
-	TESTING_FEATURES     = "staking,stargate,iterator,cosmwasm_1_1,cosmwasm_1_2"
```

Comment:
We need to rename this to capabilities (see https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES.md for more on that)

Reply:
https://github.com/CosmWasm/wasmvm/pull/414